emails/sns.py

```diff
@@ -0,0 +1,95 @@
+import base64
+import logging
+import pem
+import six
+from urllib.request import urlopen
+
+from OpenSSL import crypto
```

Comment:
do we really want to use `pyOpenSSL` directly? is there a Django wrapper on top? or another nicer library?